src/components/VAudioDetails/VRelatedAudio.vue

```diff
@@ -13,36 +16,37 @@
       />
       <LoadingIcon v-show="fetchState.isFetching" />
     </div>
-    <p v-show="!!fetchState.isError">
+    <p v-show="!!fetchState.fetchingError">
       {{ $t('media-details.related-error') }}
     </p>
   </aside>
 </template>
 
-<script>
-import { computed } from '@nuxtjs/composition-api'
+<script lang="ts">
+import { computed, defineComponent, PropType } from '@nuxtjs/composition-api'
 
 import { isMinScreen } from '~/composables/use-media-query'
+import { FetchState } from '~/composables/use-fetch-state'
+import { AudioDetail } from '~/models/media'
```

Comment:
For FectchState and AudioDetail:
> This import is never used as a value and must use 'import type' because 'importsNotUsedAsValues' is set to 'error'. Vetur(1371)

Reply:
+1, this setting is also present in the `@vue/tsconfig` [recommended defaults](https://github.com/vuejs/tsconfig/blob/26fc528db347dbbbbaa2ea15d3c2d95c197887b7/tsconfig.json#L19-L23).